dxp/build_process.md

```diff
@@ -0,0 +1,67 @@
+# DXP Build Process
+
+This article describes how the build process works in DXP, with a high-level view that provides a deeper understanding of what goes on behind the scenes, this article does not describe the fine details and there are many links to other articles that discuss in more detail parts of the process.
+
+## Build module
+
+The build pipeline of the source code of the modules in DXP is made by a set of tools that are concentrated in the CLI [liferay-npm-scripts](https://github.com/liferay/liferay-npm-tools/tree/master/packages/liferay-npm-scripts) that abstracts some configurations that would be necessary to perform the build, think of it as a zero-config to start to work.
+
+> `liferay-npm-scripts` does not only deal with the build but also with [formatting](./formatting.md), storybook, theme, test...
+
+![An overview of the build](../images/liferay-npm-scripts-pipeline.png)
+
+When running `liferay-npm-scripts build` a sequence of execution of other tools takes place, depending on the environment in which it is being run, will trigger other tools (e.g. Webpack and Soy).
```

Comment:
[webpack](https://webpack.js.org/) is one of those projects that has decided to not use an initial capital letter in its name (even at the start of a sentence, the damn hipsters), so you should sweep through and update references "Webpack" → "webpack"